### ISSUE TITLE
Add error checking in the svzerodsolver application

### ISSUE THREAD

**Issue by ktbolt** [StanfordCBCL/svZeroDPlus#38]
Running the `svzerodsolver` application without any arguments produces a segmentation fault. 

Running the `svzerodsolver` application with an input JSON file that does not exist produces
```
libc++abi.dylib: terminating with uncaught exception of type nlohmann::json_abi_v3_11_2::detail::parse_error: [json.exception.parse_error.101] parse error at line 1, column 1: syntax error while parsing value - unexpected end of input; expected '[', '{', or a literal
```

I will add checks to the `svzerodsolver` application to handle both of these cases. 

I will also add a `--help` command-line option that most applications support.

**Comment by mrp089** [comment]
@ktbolt, seems like this was addressed in #67?

**Comment by ktbolt** [comment]
@mrp089 Right you are, l'll close this.
